src/SecretsFoundry.ts

```diff
@@ -1,6 +1,9 @@
 import dotenv from 'dotenv';
 import path from 'path';
 import Loader from './loaders/loader';
+import batchingTSort from 'batching-toposort';
+import _ from 'lodash';
```

Comment:
This will include the entire lodash lib in the bundle, which is not necessary at all. Is it possible for us to either
1. Not include lodash
2. Import the necessary functions from lodash in a tree shake-able manner?